### PR TITLE
Update kube-enforcer helm chart docs

### PR DESCRIPTION
**Pull request by VineethReddy02**
These changes are suggested as enhancements to docs.

@eranbibi @niso120b

### CODE REVIEW

File: kube-enforcer/values.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 imageCredentials:
   # If aqua-registry already exists in the cluster. Make create to false. So it won't attempt to create a new registry secret. 
   create: true
-  name: csp-registry-secret # example
+  name: aqua-registry-secret # example
```
Comment (agilgur5) [inline kube-enforcer/values.yaml:5]: duplicates #110

File: kube-enforcer/README.md [hunk 2]
```diff
@@ -71,21 +81,25 @@ Optional flags:
...
+
 ## Configurable Variables
 
 ### KubeEnforcer
 
 | Parameter                         | Description                          | Default                                                                      |
 | --------------------------------- | ------------------------------------ | ---------------------------------------------------------------------------- |
 | `imageCredentials.create`               | Set if to create new pull image secret    | `true`                                                                 |
-| `imageCredentials.name`               | Your Docker pull image secret name    | `aqua-image-pull-secret`                                                                   |
+| `imageCredentials.name`               | Your Docker pull image secret name    | `aqua-registry-secret`                                                                   |
```
Comment (agilgur5) [inline kube-enforcer/README.md:95]: duplicates #110

File: kube-enforcer/README.md [hunk 3]
```diff
@@ -71,21 +81,25 @@ Optional flags:
 --aquaSecret.kubeEnforcerToken           default to "" you can find the KubeEnforcer token from aqua csp under enforcers tab in default/custom KubeEnforcer group or you can manually approve KubeEnforcer authentication from aqua CSP under default/custom KubeEnforcer group in enforcers tab.
 ```
 
+## ClusterRole
+
+KubeEnforcer needs a dedicated clusterrole with **get, list, watch** permissions on **pods, secrets, nodes, namespaces, deployments, replicasets, replicationcontrollers, statefulsets, daemonsets, jobs, cronjobs, clusterroles, clusterrolebindings, componentstatuses** to perform discovery on the cluster.
```
Comment (agilgur5) [inline kube-enforcer/README.md:86]: This Chart already contains a `ClusterRole`, I would find this very confusing; this makes it sound like I need to add a `ClusterRole` myself.

I've also made a bunch of PRs to fix some very confusing and buggy inconsistencies and this adds a new inconsistency, the Server Chart also has [a `ClusterRole`](https://github.com/aquasecurity/aqua-helm/blob/7ec21a0f3f4828c82dc1df20cfa97dcc8faafbc2/server/templates/rbac.yaml#L28), but nothing is mentioned there...

It also doesn't say why it needs each of those permissions.

**Comment by VineethReddy02** [comment]
Hi @agilgur5 
The changes made in this PR are raised in the internal mail chain and I made changes accordingly the gaps mentioned to align with CSP helm chart. More information can be provided by @eranbibi @niso120b and Andreas Lambrecht (SA team).

**Comment by agilgur5** [comment]
> I made changes accordingly the gaps mentioned to align with CSP helm chart

I'm not sure what you mean because per my in-line comment the `ClusterRole` doc addition is _not_ consistent with the CSP chart. I made a lot of PRs to fix consistency so I can say that inconsistencies are widespread and sometimes buggy between _all_ the charts.

> The changes made in this PR are raised in the internal mail chain

Ok well I only commented on what duplicated my existing PRs and the `ClusterRole` doc that I'm saying first-hand is confusing, inconsistent, and doesn't add new details (such as on _why_ which is what someone looking for docs on `ClusterRole` would look for). I'm not the only user that's found this repo incredibly difficult to use (see the issues and feedback from CS), but you don't have to take mine or anyone's feedback if you don't want to

**Pull request merged**